routes/routes.go

```diff
@@ -35,13 +35,14 @@ func Setup(_ *config.Config, r *mux.Router, hc http.HandlerFunc, interactivesHan
 		}).
 		Methods(http.MethodGet).
 		Handler(interactivesHandler)
+	// only resource_id - redirect
 	r.StrictSlash(true).
 		PathPrefix(getPath(false, false)).
 		MatcherFunc(func(r *http.Request, rm *mux.RouteMatch) bool {
 			return !strings.HasSuffix(r.URL.Path, EmbeddedSuffix)
 		}).
 		Methods(http.MethodGet).
-		Handler(interactivesHandler)
+		Handler(redirectHandler)
```

Comment:
sorry nimit - missed this - i think you wanna redirect the last route too - the embed one with just resource_id 👍